frontend/src/tests/components/commons/InboxButton.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { shallow } from "enzyme";
+import InboxButton from "../../../components/commons/InboxButton";
+
+const emailStub = {
+  id: 1,
+  to: "To 1",
+  from: "From 1",
+  subject: "Subject 1",
+  date: "Date 1",
+  body: "Body 1"
+};
+
+const emptyFunc = function(arg) {};
+
+it("renders title, description, left button and right button titles", () => {
+  const wrapper = shallow(<InboxButton email={emailStub} clickFunction={emptyFunc} />);
```

Comment:
Instead of emptyFunc, you could have `const wrapper = shallow(<InboxButton email={emailStub} clickFunction={() => {}}`

Reply:
fixed in c25d068